exercises/basic/README.md

```diff
@@ -173,6 +173,14 @@ these instances:
 make stop
 ```
 
+## The Use of Gateway (gw) and ARP Commands in Topology.json
+- Gateway (gw) Command: The `route add default gw` command is used to set the `default` gateway for a host. This tells the host which `IP address` to send packets to if the destination IP is not on the same subnet. It's important for hosts to know their default gateway to communicate with devices outside their local network.
+- ARP Command: The `arp -i eth0 -s` command is used to add `static ARP` entries to the host's ARP cache. When you add static ARP entries, you're telling your computer something like i.e. "Hey, I already know who lives at this IP address (like 10.0.0.1), so you don't need to keep asking everyone on the network 'Who has this IP?' all the time."
```

Comment:
I would recommend adding a mention that not only is this for efficiency of avoiding ARP messages, but because the switches in this exercise (and most of the other exercises) do NOT respond to ARP requests.  Production switches in real deployed networks typically do, but they do not in these exercises.  Thus the static ARP entry for the gateway router is necessary in these exercises.